frontend/src/components/workout plan/MuscleGroupSelector.tsx

```diff
@@ -61,8 +62,11 @@ const MuscleGroupSelector: React.FC<MuscleGroupSelectorProps> = ({
   };
 
   return (
-    <Dialog defaultOpen={!Boolean(value)}>
-      <DialogTrigger className="w-[180px] border hover:border-secondary-foreground rounded py-1 px-2">
+    <Dialog open={!Boolean(value) || open}>
```

Comment:
there is a bug that you cannot cancel selecting a muscle group. you have to select one. You cant close it or exit otherwise